### ISSUE TITLE
Need Chinese translation

### ISSUE THREAD

**Issue by learningpro** [datalens-tech/datalens#25]
Will you provide Chinese translation ? Or how could I make a pull request on translation

**Comment by learningpro** [comment]
https://github.com/datalens-tech/datalens-ui/pull/40
Is any one here can help approve this PR?

**Comment by learningpro** [comment]
> Hi! Thank you for your contribution.
> 
> I afraid we can't just merge that PR right now since both our i18n tools and UI components are currently not suitable for Chinese. App probably just won't be able to start.
> 
> We need some time to analyze this task and, if needed, adapt our tools.

I read through the code and found that gravity-ui doesn't have cn language, it only has en and ru, maybe we should update gravity-ui/i18n first. https://github.com/gravity-ui/i18n/blob/main/src/index.ts

**Comment by resure** [comment]
> I read through the code and found that gravity-ui doesn't have cn language, it only has en and ru, maybe we should update gravity-ui/i18n first. https://github.com/gravity-ui/i18n/blob/main/src/index.ts

Yes, that's the plan — add support for additional languages in gravity i18n, uikit, etc.

We are discussing that with our colleagues (maintainers of gravity libraries) right now, we'll link here related tasks soon.

**Comment by resure** [comment]
Related issue in Gravity UIKit: https://github.com/gravity-ui/uikit/issues/1065
